src/authorization/axiosWithAuth.js

```diff
@@ -7,7 +7,7 @@ const axiosWithAuth = () => {
       headers: {
          Authorization: token
       },
-      baseURL: 'https://character-randomizer-backend.herokuapp.com/api/'
```

Comment:
Nit: I would say change this to a process.env so that if you want to deploy it somewhere you just need a process environment variable to change to have it work on the deployed site instead of locally